### PR TITLE
feat(nc-gui): add LanguageAlias enum

### PR DESCRIPTION
**Pull request by bcakmakoglu**
## Change Summary

* Fall back to EN if language cannot be matched to `Language` or `LanguageAlias`
	* Use aliased language if match is found

## Change type

- [x] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Review by pranavxc — APPROVED** [review]

File: packages/nc-gui/lib/enums.ts [hunk 1]
```diff
@@ -55,6 +55,10 @@ export enum Language {
   'zh-Hant' = '繁體中文',
 }
 
+export enum LanguageAlias {
+  zn_CN = 'zh-Hans',
```
Comment (pranavxc) [inline packages/nc-gui/lib/enums.ts:59]: What about `zh-Hant`?

Comment (bcakmakoglu) [reply packages/nc-gui/lib/enums.ts:59]: Well, I can't assign zh_CH to two languages so I just picked the first one 🥴

Comment (pranavxc) [reply packages/nc-gui/lib/enums.ts:59]: From the git history I found the following   

```
zh_Hans = '大陆简体',
zh_Hant = '臺灣正體',
```

```
zh_CN = '大陆简体',
zh_TW = '臺灣正體',
```

Comment (bcakmakoglu) [reply packages/nc-gui/lib/enums.ts:59]:
```ts
export enum LanguageAlias {
  zn_CN = 'zh-Hans',
  zh_TW = 'zh-Hant',
}
```

So this?

Comment (pranavxc) [reply packages/nc-gui/lib/enums.ts:59]: Yes 👍

**Review by pranavxc — APPROVED** [review]

**Pull request merged**